pkgs/tools/security/ghidra/bin.nix

```
@@ -0,0 +1,70 @@
+{ stdenv, fetchzip, lib, makeWrapper, autoPatchelfHook
+, openjdk11, pam, makeDesktopItem, icoutils
+}: let
+
+  pkg_path = "$out/lib/ghidra";
+
+  desktopItem = makeDesktopItem {
+    name = "ghidra";
+    exec = "ghidra";
+    icon = "ghidra";
+    desktopName = "Ghidra";
+    genericName = "Ghidra Software Reverse Engineering Suite";
+    categories = "Development;";
+  };
+
+
+in stdenv.mkDerivation rec {
+
+  pname = "ghidra";
+  version = "9.2";
+  versiondate = "20201113";
+
+  src = fetchzip {
+    url = "https://www.ghidra-sre.org/ghidra_${version}_PUBLIC_${versiondate}.zip";
+    sha256 = "0lcvmbq04qkdsf0bz509frgw79bhyxyixkqg1k712p3576ng3nby";
+  };
+
+  nativeBuildInputs = [
+    makeWrapper
+    autoPatchelfHook
+    icoutils
+  ];
+
+  buildInputs = [
+    stdenv.cc.cc.lib
+    pam
+  ];
+
+  dontStrip = true;
+
+  installPhase = ''
+    mkdir -p "${pkg_path}"
+    mkdir -p "${pkg_path}" "$out/share/applications"
+    cp -a * "${pkg_path}"
```

```suggestion
    cp -a . "${pkg_path}"
```

Isn't this the same with the difference that yours fails if the path contains 1024 chars of files?